### PR TITLE
Update TestNG to version 7

### PR DESCRIPTION
**Pull request by wendigo**
Trino is only using TestNG in product tests so this change is contained and should not cause issues

### CODE REVIEW

File: tempto-examples/docker/ssh/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,10 @@
 FROM ghcr.io/trinodb/testing/centos7-oj11:53
```
Comment (kokosing) [inline tempto-examples/docker/ssh/Dockerfile:1]: The below is workaround, can we move to almalinux instead?

File: pom.xml [hunk 2]
```diff
@@ -42,6 +42,7 @@
         <air.check.skip-pmd>true</air.check.skip-pmd>
 
         <project.build.targetJdk>21</project.build.targetJdk>
+        <dep.testng.version>7.10.2</dep.testng.version>
```
Comment (kokosing) [inline pom.xml:45]: Does it work with Trino?

File: tempto-examples/docker/ssh/Dockerfile (hunk 1 above)
Comment (wendigo) [reply tempto-examples/docker/ssh/Dockerfile:1]: Once we release new images, sure!

**Review by losipiuk — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (losipiuk) [reply pom.xml:45]: Trino should not care is it does not use testng

Comment (kokosing) [reply pom.xml:45]: It does use. As testng is part of product tests. So maybe I should use different words. Does product tests are passing in Trino with this change? I guess we will find out after the release :)

File: tempto-examples/docker/ssh/Dockerfile (hunk 1 above)
Comment (kokosing) [reply tempto-examples/docker/ssh/Dockerfile:1]: 👍

**Review by kokosing — APPROVED** [review]

**Pull request closed without merging**